### ISSUE TITLE
[M0-Dev] Atributos de la clase "Recomendación"

### ISSUE THREAD

**Issue by manujurado1** [ignaciotitos/IV---HealthForAll#14]
A raíz de lo discutido en #11 , se ha sacado la conclusión de que es necesaria una clase "Recomendación". Por lo que:

- ¿Sería ese nombre el apropiado para la clase?
- ¿Qué atributos debería tener? Lo único que se me ocurre es un % de recomendación, pero no sé que características son las que hacen a una moneda +/- recomendable

**Comment by ignaciotitos** [comment]
La recomendación va a ser en que invertir o si ahorrar depende de la variación de la moneda.

**Comment by manujurado1** [comment]
Vale, pero, ¿eso se medirá en un %o cómo se tiene pensado medirlo? Es que según la información que me das, yo haría que la clase Recomendación tuviera un atributo para hacer referencia a la moneda y luego otros 2 atributos, uno con el % de recomendación de inversión y otro con el % de recomendación de ahorro. A esto quizás añadirle otro atributo que represente la variación de la moneda.
Si lo ves diferente, hazme saber tu punto de vista sobre como tienes pensado que esté formada esta clase

**Comment by ignaciotitos** [comment]
Sí, la verdad que había pensado medirlo de otra manera, pero veo que por porcentajes o pesos es la mejor forma de hacerlo

**Comment by manujurado1** [comment]
Vale, todo claro, en 1/2 días espero tener el PR con la primera aproximación sobre todo lo que hemos discutido. Muchas gracias!

**Comment by ignaciotitos** [comment]
Perfecto, gracias a ti!

**Comment by JJ** [comment]
Tampoco está formulado como un problema, lo que lleva a que cualquier cosa que se haga (que sean "atributos de la clase")  cierre el issue sin ningún intento de modelizar el problema.
